admin/src/components/ReactIconsSelector/index.tsx

```diff
@@ -11,35 +10,48 @@ import {
   Select,
   FieldAction,
   Option,
-  TextInput,
 } from "@strapi/design-system";
+import { TextInput as OriginalTextInput } from "@strapi/design-system";
+import styled from "styled-components";
 import * as ReactIcons from "react-icons/all";
-import { Cross } from "@strapi/icons";
-import { SearchIcon } from "@strapi/icons";
-import { useIntl, MessageDescriptor } from "react-intl";
-import { request } from "@strapi/helper-plugin";
+import { useIntl } from "react-intl";
+import { useFetchClient } from "@strapi/helper-plugin";
 import { IconContext } from "react-icons/lib";
-
-interface IReactIconsSelector {
-  description: null | MessageDescriptor;
-  intlLabel: null | MessageDescriptor;
-  placeholder: null | MessageDescriptor;
-  name: string;
-  error: string;
-  required: boolean;
-  onChange: any;
-  value: string;
-}
-
-interface IIconComponent {
-  icon: string;
-  size?: number;
-}
+import { FixedSizeGrid as Grid } from "react-window";
 
 type IReactIcon = keyof typeof ReactIcons;
 
 const strapiTheme = window.localStorage.STRAPI_THEME;
 
+const inputButtonStyle = {
+  background: "#0021a4",
```

Comment:
I don't really like this blue design... Did you see my changes in commit [7c7776fb554d95cc58537f445177549c5d3b205a](https://github.com/DanielPantle/strapi-plugin-react-icons/commit/7c7776fb554d95cc58537f445177549c5d3b205a) to fix the dark mode issue? With this, the icons are visible also in dark mode and I think it is more complient with the strapi design system.